client/app/cloud/cloud.js

```diff
@@ -1,15 +1,17 @@
 /**
  * Special rules for redirections
  */
-angular.module("managerApp").run(function ($rootScope, $state, $stateParams) {
-    $rootScope.$on("$stateChangeStart", (evt, to, params) => {
+angular.module("managerApp").run(($transitions, $state, $stateParams) => {
+    $transitions.onStart({}, transition => {
+        const to = transition.to();
+        const params = transition.params();
         if (to.redirectTo) {
```

Comment:
(optional) I think redirectTo already exists in latests versions of ui-router: https://ui-router.github.io/ng1/docs/latest/interfaces/state.statedeclaration.html#redirectto